collectors/python.d.plugin/retroshare/retroshare.chart.py

```diff
@@ -7,33 +7,33 @@
 
 from bases.FrameworkServices.UrlService import UrlService
 
-# default module values (can be overridden per job in `config`)
-# update_every = 2
-priority = 60000
 
-# charts order (can be overridden if you want less charts, or different order)
-ORDER = ['bandwidth', 'peers', 'dht']
+ORDER = [
+    'bandwidth',
+    'peers',
+    'dht',
+]
 
 CHARTS = {
     'bandwidth': {
-        'options': [None, 'RetroShare Bandwidth', 'kB/s', 'RetroShare', 'retroshare.bandwidth', 'area'],
+        'options': [None, 'RetroShare Bandwidth', 'KiB/s', 'RetroShare', 'retroshare.bandwidth', 'area'],
```

Comment:
This should be `'Kib' or 'Kibit', not 'KiB'.

Reply:
We should probably double check what Retroshare is actually exposing.  If the raw values are reported as kilobits, then it's probably using SI units not IEC units (bandwidth seems to be one of the few places SI remains dominant, probably because of the Ethernet standards).